app/templates/partials/widgets/dropdown_widget.html

```diff
@@ -0,0 +1,26 @@
+{% set label = {
+    "for": answer.id,
+    "text": answer.label,
+    "description": answer.description
+    } %}
+
+{% include 'partials/forms/label.html' %}
+
+<select class="input input--select" id="{{answer.id}}" name="{{answer.id}}">
```

Comment:
This should use the template in `app/templates/partials/forms/select.html` rather than duplicating it here.